src/platforms/php/guides/symfony/config.mdx

```diff
@@ -64,6 +59,22 @@ All the possible configurations under the `options` key map directly to the corr
 you can read more about those in the [general configuration docs](../configuration/),
 and on the [PHP specific SDK docs](../config/).
 
+If you use Monolog, you can use it to report events instead of the normal error listener approach. To do so, you must turn `register_error_listener` to `false` and configure our `Handler` as a service, like this:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
```

Comment:
think we should here again going on the symfony coding standard and use `4 spaces` in yaml files like we did it in the other doc

Reply:
Done